rocAL/rocAL_pybind/amd/rocal/types.py

```diff
@@ -77,6 +77,11 @@
 from rocal_pybind.types import GAUSSIAN_INTERPOLATION
 from rocal_pybind.types import TRIANGULAR_INTERPOLATION
 
+#      Rocal External Source Mode
```

Comment:
Please remove the additional spaces before  Rocal External Source Mode
# Rocal External Source Mode

Reply:
Was done to maintain spacing like other imports comments. Hence retaining it as it
#     RocalResizeInterpolationType
from rocal_pybind.types import NEAREST_NEIGHBOR_INTERPOLATION
from rocal_pybind.types import LINEAR_INTERPOLATION
from rocal_pybind.types import CUBIC_INTERPOLATION
from rocal_pybind.types import LANCZOS_INTERPOLATION
from rocal_pybind.types import GAUSSIAN_INTERPOLATION
from rocal_pybind.types import TRIANGULAR_INTERPOLATION

#      Rocal External Source Mode
from rocal_pybind.types import EXTSOURCE_FNAME
from rocal_pybind.types import EXTSOURCE_RAW_COMPRESSED
from rocal_pybind.types import EXTSOURCE_RAW_UNCOMPRESSED